### DISCUSSION TITLE
哔哩孪生兄弟 “哩哩”正式在github上线

### DISCUSSION THREAD

**Discussion by Q-Tai-mu** [Richasy/Bili.Uwp#387] [常规]
哩哩正式在github上线

![liliMain](https://user-images.githubusercontent.com/58099119/135859286-9e1b539b-232b-40bc-a661-eef5afc46210.png)
![lili](https://user-images.githubusercontent.com/58099119/135859295-2556639e-8ab6-409b-8c8d-05417c3c18ed.png)

> **Up主页面**

![liliUpHome](https://user-images.githubusercontent.com/58099119/136344379-107cf96e-7cc9-4850-8560-8f99fdf5211c.png)

> **其他相关页面**
![liliT (2)](https://user-images.githubusercontent.com/58099119/137274110-1379a8f1-4296-48e0-94f4-f5b8028595e3.png)

![liliBan](https://user-images.githubusercontent.com/58099119/137274124-9fb553d2-9a9a-4fe7-916b-8e9f0e8c8a0e.png)
![liliT](https://user-images.githubusercontent.com/58099119/137274136-3ef989c2-e733-4a5a-a3e5-1ac549410765.png)
![liliUpPlay](https://user-images.githubusercontent.com/58099119/137274180-e4827522-1f84-46a2-939b-2371df49d7b9.jpg)

**Comment by xvzhenduo** [comment]
？仓库地址，发一下

**Reply by Q-Tai-mu** [reply]
https://github.com/Q-Tai-mu/bilibili.electron

**Comment by ghost** [comment]
这是苹果的吧

**Reply by Q-Tai-mu** [reply]
windows

**Reply by IR141** [reply]
为啥文件没了

**Reply by Q-Tai-mu** [reply]
要自己编译

**Reply by billowssun** [reply]
没学过编程，不给用了咯？

**Reply by Michaelxia6** [reply]
求发包

**Comment by gaoysz** [comment]
这个要用什么编译器编译

**Comment by Michaelxia6** [comment]
求发个包吧

**Comment by hujiadfr** [comment]
求写个编译方法，想在linux用

**Comment by Michaelxia6** [comment]
不发包就是半成品
